modules/33acrossIdSystem.js

```diff
@@ -9,26 +9,34 @@ import { logMessage, logError } from '../src/utils.js';
 import { ajaxBuilder } from '../src/ajax.js';
 import { submodule } from '../src/hook.js';
 import { uspDataHandler } from '../src/adapterManager.js';
+import { getStorageManager } from '../src/storageManager.js';
 
 const MODULE_NAME = '33acrossId';
+const EXT_KEY = `${MODULE_NAME}_ext`;
 const API_URL = 'https://lexicon.33across.com/v1/envelope';
 const AJAX_TIMEOUT = 10000;
 const CALLER_NAME = 'pbjs';
+const GVLID = 58;
 
-function getEnvelope(response) {
+export const storage = getStorageManager({ gvlid: GVLID, moduleName: MODULE_NAME });
+
+function calculateResponseObj(response) {
   if (!response.succeeded) {
     logError(`${MODULE_NAME}: Unsuccessful response`);
 
-    return;
+    return {};
   }
 
   if (!response.data.envelope) {
     logMessage(`${MODULE_NAME}: No envelope was received`);
 
-    return;
+    return {};
   }
 
-  return response.data.envelope;
+  return {
+    envelope: response.data.envelope,
+    ext: { ...response.data.ext }
```

Comment:
What is the value of shallow clone? Why not a deep clone if you want immutability? If not, why not just assign `response.data.ext` directly?

Reply:
Also note that at this point `ext` in the calculated response will contain `eids.rubicon:{...}` only.

Reply:
that's true, it doesn't make sense to clone it.. will change it.

and yes, I am aware that it's going to be only rubicon.